### PR TITLE
issue-1-create-an-entry - Daniel Pham

### PR DESCRIPTION
**Pull request by dnypham**
https://user-images.githubusercontent.com/85265067/132574480-391e4fdd-479f-48a0-947c-f4299ab7badc.mp4

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1,2 +1,27 @@
...
+  }
+  if (event.target.matches('.photo-url') && event.target.value === '') {
+    $img.setAttribute('src', 'images/placeholder-image-square.jpg');
+  }
+});
+
+$form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var entry = {
+    title: $form.title.value,
```
Comment (scbowler) [inline js/main.js:18]: You need to set an `entryId` using `data.nextEntryId`

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just missed setting the `entryId`. Let me know if you have any questions.

File: js/main.js (hunk 1 above)
Comment (dnypham) [reply js/main.js:18]: Screen Shot 2021-09-08 at 12 53 05 PM

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**